docs/3_data_simulation.rst

```diff
@@ -0,0 +1,15 @@
+Data Simulations
+==================
+Usually, public-health-related data can not be disclosed publicly which makes the development and systematic comparison of outbreak detection algorithms hard. Luckily, using simulations of hypothetical disease spreads is a valid alternative to real data. Thus, epysurv includes a module that allows you to simulate simple, univariate endemic and epidemic timeseries, i.e., with and without outbreak.
```

Comment:
Is there maybe a review paper that backs some of the claims made here that we could reference, as in the outbreak detection section?